mmv1/templates/terraform/examples/dataproc_metastore_service_dpms2.tf.erb

```diff
@@ -0,0 +1,21 @@
+resource "google_dataproc_metastore_service" "<%= ctx[:primary_resource_id] %>" {
+  service_id = "<%= ctx[:vars]['metastore_service_name'] %>"
+  provider   = google-beta
```

Comment:
This should only be present if the field is beta-only

Reply:
Thanks, resolved in commit [7b02a51](https://github.com/GoogleCloudPlatform/magic-modules/pull/8526/commits/7b02a5191f51da11cda4145875a5b4f41750ef8c).